frontend/src/app/main/ui/workspace/colorpicker.cljs

```diff
@@ -225,15 +227,15 @@
        [:div.colorpicker-tab.tooltip.tooltip-bottom.tooltip-expand
         {:class (when (= @active-tab :ramp) "active")
          :alt (tr "workspace.libraries.colors.rgba")
-         :on-click set-ramp-tab!} i/picker-ramp]
+         :on-click (partial set-tab! :ramp)} i/picker-ramp]
```

Comment:
this is an anti-performance change, the tab name should be passed as attribute

Reply:
Thanks!, fixed! :)